service/github/github.go

```diff
@@ -418,11 +441,11 @@ func buildBody(c *reviewdog.Comment, baseRelatedLocURL string, gitRootPath strin
 		}
 		cbody += "\n<hr>\n\n" + relatedLoc.GetMessage() + "\n" + relatedURL
 	}
-	cbody += fmt.Sprintf("\n<!-- __reviewdog__:%s -->\n", buildMetaComment(fprint, c.ToolName))
+	cbody += fmt.Sprintf("\n<!-- __reviewdog__:%s -->\n", BuildMetaComment(fprint, toolName))
 	return cbody
 }
 
-func buildMetaComment(fprint string, toolName string) string {
+func BuildMetaComment(fprint string, toolName string) string {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported function BuildMetaComment should have comment or be unexported
<!-- __reviewdog__:ChBlMjc2M2E0NjljMDVlYzYzEg9nb2xpbnQtY2lyY2xlY2k= -->